dspot/src/main/java/eu/stamp_project/utils/collector/CollectorConfig.java

```diff
@@ -21,6 +21,8 @@ public static CollectorConfig getInstance() {
 
     /* Variables and getters/setters */
     private static DspotInformationCollector collector = new NullCollector();
+    private static String gmailUserName;
```

Comment:
we don't want to depend on GMail, it should be `smtpUserName`

Reply:
Now gmail has become the more general smtp :) .

Reply:
all variables called `gmail*` should be renamed `smtp*`

Reply:
Done :) . Those are redundant lines . I forgot to remove them.